nginx/tests/test_unit.py

```diff
@@ -36,10 +36,11 @@ def test_flatten_json_timestamp(check):
     )
 
 
-def test_plus_api(check, instance, aggregator):
+def test_plus_api_v2(check, instance, aggregator):
     instance = deepcopy(instance)
     instance['use_plus_api'] = True
     instance['use_plus_api_stream'] = True
+    instance['plus_api_version'] = 2
```

Comment:
Quick question: do we need this explicit setting? I see that by default we use `plus_api_version: 2`

https://github.com/DataDog/integrations-core/blob/c2add22bb30a6fc0ebe9e7bc297837d1b840b6d3/nginx/datadog_checks/nginx/nginx.py#L164

Also asking because if users of API v2 end up having to explicitly set the version then it might be a breaking change. (Just want to confirm that this feature is being added respecting backwards compatibility.)

Does it work as expected if we remove this line…?

```suggestion
```

Reply:
yes we still use default version 2. The purpose of the line was more to help the developer better understand that v2 is actually being used. If this line is deleted, we will have the same behavior. Should I remove or keep?

Reply:
OK, understood, just wanted to make sure this wasn't added because otherwise it wouldn't work. Thanks!